src/main/java/com/minecolonies/coremod/network/messages/server/colony/building/sifter/SifterSettingsMessage.java

```diff
@@ -47,33 +33,23 @@ public SifterSettingsMessage()
      * @param building      the building to set it for.
      * @param dailyQuantity the quantity to produce.
      * @param block         the mode to set.
-     * @param mesh          the mesh.
-     * @param buy           if its a buy action.
      */
-    public SifterSettingsMessage(@NotNull final BuildingSifter.View building, final ItemStorage mesh, final int dailyQuantity, final boolean buy)
+    public SifterSettingsMessage(@NotNull final BuildingSifter.View building,final int dailyQuantity)
```

Comment:
why remove this space?
![image](https://user-images.githubusercontent.com/63612548/110516331-1131e400-80cf-11eb-8bd8-201f40bd9985.png)

Reply:
Accidental extra space removal when removing the extra params